pynetworktables2js/js/networktables.js

```diff
@@ -282,7 +282,7 @@ const NetworkTables = new function () {
 			};
 			
 			socket.onmessage = function(msg) {
-				const data = JSON.parse(msg.data);
+				var data = CBOR.decode(msg.data);
```

Comment:
Did you mean to change this back to `var`?

Reply:
Nice catch, I'll fix that